core/templates/filters/translate.pipe.ts

```diff
@@ -90,9 +90,9 @@ export class TranslatePipe implements PipeTransform, OnDestroy {
 
   /**
    * Every pipe is supposed to have a function named transform as Pipes
```

Comment:
... named "transform". This is because Pipes implement the PipeTransform interface. ...

(Otherwise, everything looks good. Please note that PipeTranform is still misspelled in your most recent commit. Please be sure to proofread your changes carefully -- maybe using a diff viewer like `meld` -- before pushing.)

Reply:
Changed!